docs/src/main/asciidoc/property-source-config.adoc

```diff
@@ -251,8 +251,9 @@ When enabled, the `SecretsPropertySource` looks up Kubernetes for `Secrets` from
 
 Note that, by default, consuming Secrets through the API (points 2 and 3 above) *is not enabled* for security reasons.
 Further, we recommend that containers share secrets through mounted volumes.
-If you enable consuming Secrets through the API, we recommend that you limit access to Secrets by using an
-[authorization policy, such as RBAC](https://kubernetes.io/docs/concepts/configuration/secret/#best-practices).
+If you enable consuming Secrets through the API, we recommend that you limit access to Secrets by using an authorization policy, such as RBAC. 
```

Comment:
Perhaps we can also add a note that there is no way to limit access to specific secrets inside a Kubernetes namespace with RBAC - meaning that am app that can read a secret in a namespace can read all of them?

Reply:
It is mentioned in the referenced link but we can also add it here

Reply:
Since it's rather serious, I propose we mention it here as well